Makefile.am: remove doc

There was a failure at do_install time, so remove doc to workaround.

Upstream-Status:  Inappropriate workaround
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 6648eff..c6694b9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = po src tests doc
+SUBDIRS = po src tests
 
 EXTRA_DIST = \
     libreport.pc.in \
-- 
1.8.1.2

